fe/fe-core/src/main/java/org/apache/doris/statistics/OlapAnalysisTask.java

```diff
@@ -65,7 +71,25 @@ public class OlapAnalysisTask extends BaseAnalysisTask {
                     + "MIN(`${colName}`) AS min, "
                     + "MAX(`${colName}`) AS max, "
                     + "${dataSizeFunction} AS data_size, "
-                    + "NOW() FROM `${dbName}`.`${tblName}` PARTITION ${partitionName}  ${sampleExpr}";
+                    + "NOW() FROM `${dbName}`.`${tblName}` PARTITION ${partitionName}";
+
+    private static final String SAMPLE_COLUMN_SQL_TEMPLATE = "SELECT \n"
+            + "CONCAT(${tblId}, '-', ${idxId}, '-', '${colId}') AS id, \n"
+            + "${catalogId} AS catalog_id, \n"
+            + "${dbId} AS db_id, \n"
+            + "${tblId} AS tbl_id, \n"
+            + "${idxId} AS idx_id, \n"
+            + "'${colId}' AS col_id, \n"
+            + "NULL AS part_id, \n"
+            + "COUNT(1) * ${ratio} AS row_count, \n"
+            + "NDV(`${colName}`) * ${ratio}  AS ndv, \n"
+            + "SUM(CASE WHEN `${colName}` IS NULL THEN 1 ELSE 0 END) * ${ratio} AS null_count, \n"
```

Comment:
How about rename "ratio" by "scale", like "sum(...) * ${scale}"? This double always larger than 1.